app/src/main/java/org/p2p/wallet/auth/web3authsdk/GoogleSignInHelper.kt

```diff
@@ -70,6 +71,11 @@ class GoogleSignInHelper(
         }
     }
 
+    private fun isValidError(exception: ApiException): Boolean {
```

Comment:
isValidError -> shouldErrorBeHandled
that's what method intents to do, it checks should we handle error from google